### PR TITLE
chore(common): CHP-7101 convert to monorepo setup

### PR DESCRIPTION
**Pull request by chanceaclark**
## What/why?

- Moved files around in preparation for the monorepo
- Converted to a monorepo with yarn workspaces + lerna.

### CODE REVIEW

File: packages/validate-commits/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "license": "MIT",
+  "bin": {
+    "validate-commits": "./bin/validate-commits.js"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "engines": {
+    "node": ">=14.0.0",
+    "npm": ">=6.0.0"
```
Comment (chanceaclark) [inline packages/validate-commits/package.json:23]: Updated the engines per @deini's comment in #50

File: package.json [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
+    "test": "lerna run test --stream"
   },
   "engines": {
-    "node": "^14.0.0",
-    "npm": "^6.0.0"
-  },
-  "dependencies": {
-    "chalk": "^4.1.1",
-    "minimist": "^1.2.0"
+    "node": "^14.0.0"
```
Comment (chanceaclark) [inline package.json:19]: Locked the root engine to node 14 for local development.

File: package.json [hunk 3]
```diff
@@ -1,32 +1,25 @@
...
-  "description": "Validates commits based on a series of scopes/tasks",
-  "files": [
-    "bin",
-    "lib"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/bigcommerce/validate-commits.git"
-  },
+  "name": "@bigcommerce/commitlint",
```
Comment (chanceaclark) [inline package.json:2]: Lmk if you have a better name

File: packages/validate-commits/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "dependencies": {
+    "chalk": "^4.1.1",
+    "minimist": "^1.2.0"
+  },
+  "devDependencies": {}
```
Comment (deini) [inline packages/validate-commits/package.json:29]: 🍹 delete the empty `devDependencies`

**Review by deini — APPROVED** [review]

File: packages/validate-commits/package.json (hunk 4 above)
Comment (chanceaclark) [reply packages/validate-commits/package.json]: done

**Pull request merged**